Store/src/database/database.cs

```diff
@@ -33,6 +36,11 @@ public static void CreateDatabase(StoreConfig config)
             throw new Exception("Database credentials in config must not be empty!");
         }
 
+        if (!config.Settings.TryGetValue("database_equip_table_name", out equipTableName))

```

Comment:
Why do we change table name of only store_equipments? And why do we need to change it in config?

Reply:
Custom equipment database table name for preventing equipment conflicts across different servers. 
For example, ZombieEscape Mode only requires team CT, which conflicts with other servers.